.storybook/config.js

```diff
@@ -8,7 +8,16 @@ import '../plugins/vue-filters'
 import '../assets/scss/style.scss';
 
 Vue.use(BootstrapVue);
-
+Vue.prototype.$t = function () {}
```

Comment:
The entire code addition to this file needs to be linted.

Reply:
Removed, and now uses Vue-i18n. Seems to do the trick :)